.ci/shippable.sh

```diff
@@ -78,9 +78,105 @@ pitest-checks-coding)
+  "JavadocParagraphCheck.java.html:<td class='covered'><pre><span  class='survived'>            if (nextNode.getType() == JavadocTokenTypes.TEXT</span></pre></td></tr>"
+  "JavadocParagraphCheck.java.html:<td class='covered'><pre><span  class='survived'>                    || nextNode.getType() == JavadocTokenTypes.HTML_ELEMENT) {</span></pre></td></tr>"
+  "JavadocStyleCheck.java.html:<td class='covered'><pre><span  class='survived'>                    &#38;&#38; builder.charAt(index - 1) == &#39;*&#39;) {</span></pre></td></tr>"
+  "JavadocStyleCheck.java.html:<td class='covered'><pre><span  class='survived'>            else if (index &#62; 0 &#38;&#38; builder.charAt(index) == &#39;/&#39;</span></pre></td></tr>"
+  "JavadocStyleCheck.java.html:<td class='covered'><pre><span  class='survived'>                trimTail(builder);</span></pre></td></tr>"
+  "JavadocTagContinuationIndentationCheck.java.html:<td class='covered'><pre><span  class='survived'>                            &#38;&#38; (text.length() &#60;= offset</span></pre></td></tr>"
+  "JavadocTagContinuationIndentationCheck.java.html:<td class='covered'><pre><span  class='survived'>        while (inlineTag != null) {</span></pre></td></tr>"
+  "JavadocTag.java.html:<td class='uncovered'><pre><span  class='survived'>        return tagInfo == JavadocTagInfo.SEE</span></pre></td></tr>"
+  "JavadocTypeCheck.java.html:<td class='covered'><pre><span  class='survived'>                    tagCount++;</span></pre></td></tr>"
+  "SummaryJavadocCheck.java.html:<td class='covered'><pre><span  class='survived'>        for (int i = 0; !found &#38;&#38; i &#60; children.length - 1; i++) {</span></pre></td></tr>"
+  "SummaryJavadocCheck.java.html:<td class='covered'><pre><span  class='survived'>            if (child.getType() != JavadocTokenTypes.JAVADOC_INLINE_TAG</span></pre></td></tr>"
+  "TagParser.java.html:<td class='covered'><pre><span  class='survived'>                       &#38;&#38; (Character.isWhitespace(currentLine.charAt(column))</span></pre></td></tr>"
+  "TagParser.java.html:<td class='covered'><pre><span  class='survived'>                           || currentLine.charAt(column) == &#39;*&#39;)) {</span></pre></td></tr>"
+  "TagParser.java.html:<td class='covered'><pre><span  class='survived'>                while (column &#60; currentLine.length()</span></pre></td></tr>"
+  "WriteTagCheck.java.html:<td class='covered'><pre><span  class='survived'>            log(lineNo, MSG_MISSING_TAG, tag);</span></pre></td></tr>"
+  "WriteTagCheck.java.html:<td class='covered'><pre><span  class='survived'>                    tagCount += 1;</span></pre></td></tr>"
+  );
+  checkPitestReport "${ignoredItems[@]}"
+  ;;
+
+pitest-checkstyle-gui)
+  mvn -e -P$1 clean test org.pitest:pitest-maven:mutationCoverage;
+  # post validation is skipped, we do not test gui throughly
```

Comment:
will we ask users for this when they modify gui?

Reply:
no, we will not enforce pitest over gui code at all.

Reply:
Why do we even run pitest for gui then?

Reply:
good point, disabled in separate(2nd) commit, to not waste time of CI.